packages/db/src/schema.prisma

```diff
@@ -44,6 +44,8 @@ model AuthUser {
   translatorNotes    TranslatorNote[]
   /// The list of footnotes authored by this user.
   footnotes          Footnote[]
+  /// The list of gloss updates this user has performed.
+  glossHistory       GlossHistory[]
```

Comment:
I would move this before `translatorNotes`. It's more important.

Reply:
Makes sense :+1: Done.